code/modules/spells/targeted/frenchcurse.dm

```diff
@@ -0,0 +1,39 @@
+/spell/targeted/frenchcurse
+	name = "French Curse"
+	desc = "This curse will silence your target for a very long time."
+
+	school = "evocation"
+	charge_max = 300
+	invocation = "FU'K Y'U D'NY"
+	invocation_type = "shout"
+	range = 1
+	cooldown_min = 100 //100 deciseconds reduction per rank
```

Comment:
You need to reduce this as well else people will possibly have instant cooldown on it.